src/app/shared/utils.ts

```diff
@@ -28,3 +28,11 @@ export const styleVariables: any = (() => {
     return { ...styleObj, [prop.trim()]: value };
   }, {});
 })();
+
+export const filterById = (array = [], id: string) => array.filter(item => item._id !== id);
+
+export const itemsShown = (paginator: any) => Math.min(paginator.length - (paginator.pageIndex * paginator.pageSize), paginator.pageSize);
+
+export const postToDeleteItems = (dbService: any, dbName: string, deleteArray: any) => {
```

Comment:
This function should be included in CouchService directly.  Having a function which always requires the same service as the first parameter 

I think that would increase the number of functions on CouchService beyond the Code Climate limit, but I think more functions on CouchService is preferable to this.

Please call the function `bulkDocs`.

Reply:
Thanks for the advice. I will fix that.